### PR TITLE
initial working candidate nf_wochenende 

### PR DESCRIPTION
**Pull request by colindaven**
only python script working on my end so far, no additional tools/directories

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -32,12 +21,13 @@ adapter_fastp_general: /mnt/ngsnfs/seqres/contaminants/2020_02/adapters/adapters
...
+#adapter_truseq: /mnt/ngsnfs/tools/miniconda3/envs/wochenende/share/trimmomatic-0.39-2/adapters/TruSeq3-PE.fa
+adapter_truseq: /home/hpc/davenpor/programs/miniconda3/envs/wochenende/share/trimmomatic-0.39-2/adapters/TruSeq2-PE.fa
 ea_adapter_fasta: /mnt/ngsnfs/seqres/contaminants/2020_02/adapters/adapters.fa
 # Paths to tools
-path_abra_jar: /mnt/ngsnfs/tools/abra2/abra2_latest.jar
-path_afterqc: /mnt/ngsnfs/tools/afterQC/AfterQC-0.9.6/after.py
-path_alignerboost: /mnt/ngsnfs/tools/dev/Wochenende/dependencies/AlignerBoost.jar
+#path_abra_jar: /mnt/ngsnfs/tools/abra2/abra2_latest.jar
+#path_afterqc: /mnt/ngsnfs/tools/afterQC/AfterQC-0.9.6/after.py
+#path_alignerboost: /mnt/ngsnfs/tools/dev/Wochenende/dependencies/AlignerBoost.jar
```
Comment (LisaHollstein) [inline config.yaml:30]: Just a question: why isn't this needed anymore? Is this set somewhere else?

File: nf_wochenende.nf [hunk 2]
```diff
@@ -90,18 +90,24 @@ if (params.help) {
...
+    input_fastq_R2 = Channel.fromPath("*_R2.fastq", checkIfExists: false)
+
     chunksize = Channel.value(1000)
 
 
     // run processes
    
     
-    wochenende(input_fastq)
+    wochenende(input_fastq_R1, input_fastq_R2)
```
Comment (LisaHollstein) [inline nf_wochenende.nf:111]: what if the data is single end?

File: nf_wochenende.nf [hunk 3]
```diff
@@ -157,15 +164,19 @@ process wochenende {
 
     input:
     file fastq
+    file fastq2
 
 
     output:
     file "${prefix}.bam"
```
[the remark this answers is not in the document]
Comment (colindaven) [reply nf_wochenende.nf:171]: Do you mean line 167 or line 171 ?

File: config.yaml (hunk 1 above)
Comment (colindaven) [reply config.yaml:30]: No, just not using these tools at present. Abra2.jar has a security issue - log4j - and still hasn't been fixed 6 months later ...

File: nf_wochenende.nf (hunk 3 above)
[the remark this answers is not in the document]
Comment (colindaven) [reply nf_wochenende.nf:171]: Also - yes - haven't worked on the output at all really. It depends if we use a single working dir for all output from Wochenende OR pass relevant files on to raspir, haybaler etc. We need to analyze this.

[the remark this answers is not in the document]
Comment (LisaHollstein) [reply nf_wochenende.nf:171]: To answer your question: I was referring to line 171

**Review by LisaHollstein — APPROVED** [review]
Apart from the issues discussed in the conversation, this looks realy good!

**Comment by colindaven** [comment]
Thanks, please feel free to change and improve anything. There's plenty to do

**Pull request merged**